services/dumper.js

```diff
@@ -141,16 +114,16 @@ function Dumper(config) {
     fs.writeFileSync(`${pathDest}/.env`, template(settings));
   }
 
-  function writeModels(pathDest, table, fields, references) {
+  function writeModel(pathDest, table, fields, references, options) {
     const templatePath = `${__dirname}/../templates/model.txt`;
     const template = _.template(fs.readFileSync(templatePath, 'utf-8'));
 
     const text = template({
       table,
       fields,
       references,
-      underscored: isUnderscored(fields),
-      timestamps: hasTimestamps(fields),
+      underscored: options.underscored,
```

Comment:
`...options` would be even better. So if we add a new options we don't have to add it here every time ;)